app/src/main/res/values/strings.xml

```diff
@@ -14,22 +16,25 @@
     <string name="skip_button_text">Skip</string>
     <string name="text_button_onboarding_page">Next</string>
 
-    <!--    LOGIN TO REGISTER  -->
+    <!--    LOGIN TO REGISTER STRINGS  -->
```

Comment:
Same thing about region comments as in gradle
`<!-- region Authentication -->` combined with `<!-- endregio
![image](https://user-images.githubusercontent.com/84084708/191179579-80c973c7-4c98-4e6e-9bb7-dce7a4f94370.png)
![image](https://user-images.githubusercontent.com/84084708/191179595-c499507d-8fc3-45e6-85f9-0f5eb956279e.png)

n -->` will allow you to collapse regions